MetaMorpheus/MetaMorpheusSetup/MetaMorpheusSetup.wixproj

```diff
@@ -14,6 +14,12 @@
     <SuppressIces>
     </SuppressIces>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='UbuntuMac|x64'">
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
Wix builds .msi, which are specific to windows

Reply:
It needs to be defined in the csproj file, but the sln file tells it not to build wix and metamorpheus setup when running on UbuntuMac configuration

Reply:
removed this